libcontainer/cgroups/systemd/v1.go

```diff
@@ -222,7 +223,14 @@ func (m *legacyManager) Destroy() error {
 		return err
 	}
 	unitName := getUnitName(m.cgroups)
-	if err := stopUnit(dbusConnection, unitName); err != nil {
+
+	err = stopUnit(dbusConnection, unitName)
+	// Both on success and on error, cleanup all the cgroups we are aware of.
+	// Some of them were created directly by Apply() and are not managed by systemd.
+	for _, path := range m.paths {
+		unix.Rmdir(path)
```

Comment:
There is rmdir(path string) in libcontainer/cgroups/fs2/fs2.go

Can that func be used ?
It seems we should return error if the cleanup isn't successful.

Thanks

Reply:
it is a different package and rmdir is not exported.

I've added the error check.